scripts/copy-released-manifests.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+set -e
+
+SCRIPT_DIR="$(dirname "$(readlink -f "$0")")"
+REPO_DIR="$(realpath "${SCRIPT_DIR}/..")"
+
+source "${SCRIPT_DIR}/common.sh"
+
+visit "${REPO_DIR}/tasks"
+    for TASK_NAME in *; do
+        mkdir -p "../released-tasks/${TASK_NAME}/${RELEASE_VERSION}"
+        cp -r "${TASK_NAME}/." "../released-tasks/${TASK_NAME}/${RELEASE_VERSION}/"
```

Comment:
I'm not sure if it should be `vX.Y.Z` or `X.Y.Z`. I'd recommend to keep it as `X.Y.Z` if possible, because this is also what other projects do, e.g. buildah [1].

[1] https://github.com/openshift-pipelines/tektoncd-catalog/tree/p/tasks/task-buildah